helpers/suggestions/suggestion_creator.py

```diff
@@ -128,3 +129,33 @@ def createMatchVideoYouTubeSuggestion(cls, author_account_key, youtube_id, match
                 return 'video_exists'
         else:
             return 'bad_url'
+
+    @classmethod
+    def createOffseasonEventSuggestion(cls, author_account_key, name, start_date, end_date, website, address):
```

Comment:
Should we make website required? Also, make it clear in the UI which fields are required.